src/components/MovieListControl/MovieListControl.js

```diff
@@ -0,0 +1,32 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import SelectOption from '../../utils/SelectOption';
+import Dropdown from '../Dropdown/Dropdown';
+import './MovieListControl.scss';
+
+const MovieListControl = (props) => {
+  return (
+    <div className="movie-list-controls ">
+      <div className="filter-control">
+        {props.filterOptions.map((g) => (
```

Comment:
let's set the default props === [] for `filterOptions`

Reply:
Updated in 9bc0633